src/php-bufflog/BuffLog.php

```diff
@@ -2,123 +2,105 @@
 namespace Buffer;
 require_once('vendor/autoload.php');
 
-use Monolog\Logger;
-use Monolog\Handler\StreamHandler as MonologStreamHandler;
-
-/*
-    Level of logs we use:
-
-    This level require manual action to appear in Datadog Logs
-    Logger::DEBUG
-    Logger::INFO
-
-    Everything at this level appears by default in Datadog Logs
-    Logger::WARNING
-    Logger::ERROR
-    Logger::CRITICAL
-*/
+use Monolog\Logger as Logger;
+use Monolog\Handler\StreamHandler;
 
 class BuffLog {
 
-    private static $logger = null;
-    private static $currentVerbosity = Logger::WARNING;
-    private static $verbosityList = [
+    protected   static $instance;
+    private     static $logger = null;
+    private     static $currentVerbosity = Logger::WARNING;
+    private     static $verbosityList = [
         "DEBUG" =>      Logger::DEBUG,
         "INFO" =>       Logger::INFO,
         "WARNING" =>    Logger::WARNING,
```

Comment:
Should there be a 'NOTICE' line in here to match up with the `$logOutputMethods` below?

Reply:
ahh yes good spot @colinscape !
I think I should just remove that list, it's not used. I will try to see if we can control verbosity via `Monolog`